specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9538 @@
+          "description": "Indicates whether the error log is available."
+        }
+      }
+    },
+    "IngestionExecutionDetails": {
+      "description": "The ingestion execution details.",
+      "type": "object",
+      "properties": {
+        "ingestionStartTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The ingestion start time."
+        },
+        "ingestionEndTime": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The ingestion end time."
+        },
+        "status": {
+          "enum": [
+            "None",
+            "Succeeded",
+            "Queued",
+            "InProgress",
+            "SourceUnknown",
+            "PartialSucceeded",
+            "Failed",
+            "Canceled",
+            "Canceling"
+          ],
+          "type": "string",
+          "description": "The ingestion status.",
+          "x-ms-enum": {
+            "name": "IngestionPhaseStatus",
+            "modelAsString": true
+          }
+        },
+        "statistics": {
+          "type": "object",
+          "description": "The ingestion statistics.",
+          "additionalProperties": {}
+        },
+        "estimatedTimeRemainingInSeconds": {
+          "format": "int64",
+          "description": "The estimated time remaining in seconds.",
+          "type": "integer"
+        },
+        "isErrorLogAvailable": {
+          "type": "boolean",
+          "description": "Indicates whether the error log is available."
+        },
+        "lastUpdatedAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The last updated time."
+        }
+      }
+    },
+    "ScanResultWithIngestion": {
+      "description": "The scan result with ingestion.",
+      "type": "object",
+      "properties": {
+        "discoveryExecutionDetails": {
+          "$ref": "#/definitions/DiscoveryExecutionDetails",
+          "readOnly": true,
+          "description": "The discovery execution details."
+        },
+        "ingestionExecutionDetails": {
+          "$ref": "#/definitions/IngestionExecutionDetails",
+          "readOnly": true,
+          "description": "The ingestion execution details."
+        },
+        "parentId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The parent scan result identifier."
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The scan result identifier."
+        },
+        "ingestionJobId": {
+          "type": "string",
+          "description": "The ingestion job identifier."
+        },
+        "resourceId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource identifier."
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The scan result status."
+        },
```

Comment:
One more point I missed. This status field should have enum values
```suggestion
        "status": {
          "type": "string",
          "enum": [
            "Accepted",
            "InProgress",
            "TransientFailure",
            "Succeeded",
            "Failed",
            "Canceled"
          ],
          "description": "The scan result status.",
          "x-ms-enum": {
            "name": "ScanRunStatus",
            "modelAsString": true
            "values": [
              {
                "value": "Accepted",
                "description": "the scan is accepted"
              },
              {
                "value": "InProgress",
                "description": "the scan is InProgress"
              },
              {
                "value": "TransientFailure",
                "description": "the scan is in transient failure"
              },
              {
                "value": "Succeeded",
                "description": "the scan is succeeded"
              },
              {
                "value": "Failed",
                "description": "the scan is failed"
              },
              {
                "value": "Canceled",
                "description": "the scan is canceled"
              }
            ]
          }
        },
```

Reply:
Yes, changed to enum style and list values here